src/components/BooksPage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+const bookList = [
+  {
+    booktitle: ' Pride and Prejudice',
+    author: ' Jane Austen',
+    Category: ' Romance',
+    id: '1',
+  },
+  {
+    booktitle: ' The Great Gatsby',
+    author: ' F. Scott Fitzgerald',
+    Category: ' Fiction',
+    id: '2',
+  },
+];
+const BooksPage = () => (
+  <div className="books">
+    <ul className="books-list-section">
+      {bookList.map((book) => (
+        <li key={book.id}>
+          <p>
+            Book:
+            {book.booktitle}
+          </p>
+          <p>
+            Author:
+            {book.author}
+          </p>
+          <p>
+            Category:
+            {book.Category}
+          </p>
+          <button type="button">Remove</button>
+        </li>
+      ))}
+    </ul>
+    <form className="add-book-section">
+      <h1>ADD NEW BOOK</h1>
+      <input type="text" name="Book title" />
+      <select>
+        <option value="">Category</option>
+        <option value="Romance">Romance</option>
+        <option value="Documentary">Documentary</option>
+        <option value="Fiction">Fiction</option>
+        <option value="Crime">Crime</option>
+      </select>
+    </form>
```

Comment:
- [ ] In keeping with the SOLID principle, kindly extract this form for a new book into a separate component and render it in this `BookPage.js` component.
- [ ] Also add a placeholder **`button`** for adding a new book to your form. This button does not have to work yet.

Reply:
Done